src/components/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+
+import NotificationsFeed from './NotificationsFeed'
+import { ActiveTab, DCLNotification, NotificationLocale } from './types'
+
+import NotificationBell from '../Icons/Notifications/NotificationBell'
+import NotificationBellActive from '../Icons/Notifications/NotificationBellActive'
+
+import './Notifications.css'
+
+export interface NotificationsProps {
+  isOpen: boolean
+  userNotifications: DCLNotification[]
+  isLoading: boolean
+  locale: NotificationLocale
+  isOnboarding: boolean
+  activeTab: ActiveTab
+  onClickToggle: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
```

Comment:
Let's continue with the standard
```suggestion
  onClick: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
```